controller/templates/controller/partials/state_machine.html

```diff
@@ -1,6 +1,42 @@
 {% load render_table from django_tables2 %}
 <div _="on load if #argsDialog is not null remove #argsDialog"></div>
-<form>
-  {% csrf_token %}
-  {% render_table table %}
-</form>
+<div class="row">
+  <div class="col-md-auto">
+    <form>
+      {% csrf_token %}
+      {% render_table table %}
+    </form>
+  </div>
+  <div class="col-md-auto">
+    <pre class="mermaid">
+      flowchart TD
```

Comment:
I wonder if it would look nicer LR instead of TD...

Reply:
See https://github.com/ImperialCollegeLondon/drunc_ui/pull/306#issuecomment-2604757006